tests/monster_extra.fbs

```diff
@@ -1,5 +1,9 @@
 namespace MyGame;
 
+struct ArrayStruct {
+    array:[int:2];
+}
+
```

Comment:
We can remove the array from MosterExtra.
It was added before arrays_test.fbs was introduced.

Reply:
Removed.